packages/backend/jobs-manager/service/src/modules/database/custom-jobs/custom-jobs.service.ts

```diff
@@ -34,6 +34,31 @@ export class CustomJobsService {
     return r;
   }
 
+  public async duplicate(jobId: string) {
+    const existingJob = await this.customJobModel.findById(
+      new Types.ObjectId(jobId),
+    );
+
```

Comment:
Maybe un ptit `if` du genre :

```suggestion

if (!existingJob) throw new HttpNotFoundException('Job not found');

```